uploadNeuroDB/NeuroDB/MRI.pm

```diff
@@ -763,19 +762,24 @@ sub in_range
     return 0 unless $range_string;
     return 0 unless defined($value);
 
-    my @ranges = split(/,/, $range_string);
+    ## mri_protocol table is being normalized in steps. 
+    ## Until this is completed in full and/or the database 
+    ## prevents such a setting, do not allow projects
+    ## to put comma-separated values in the table columns.
+    ## If they do, LOG the specific scan type as a violated scan  
+    if($range_string=~ /,/) {
+        print STDERR "Comma separated ranges, as found in '$range_string', are no longer supported. Please remove. Logging as a violated scan. \n";
```

Comment:
I would modify the message to something like that:
"Comma separated ranges, as found in '$range_string', are no longer supported. Please modify your mri_protocol table accordingly. Logging the scan as a violated scan for now.\n"

Also, code should probably not exceed 80 characters per line so that we can view it on Github without having to scroll to the right. Very nitpicky, I know ;).

Reply:
I took your feedback into account, so I used your message but in a different place.